common/log.py

```diff
@@ -57,7 +57,10 @@ def init_logging(
     logger.info(f"Logging to {filename}")
     logger.addHandler(fh)
     logger.info("== Initializing logfile ==")
-    logger.info(f"Replay ID: {log_id}") if log_id else None
+    if script_type == 'extract':
+        logger.info(f"Extract ID: {log_id}") if log_id else None
+    else:
+        logger.info(f"Replay ID: {log_id}") if log_id else None        
```

Comment:
Could this be simplified? Just `logger.info(f"{log_id.capitalize()}: {log_id}") if log_id else None`?

Reply:
Are you looking for something like this?
``` 
if script_type == "extract":
        if log_id:
            logger.info(f"Extract ID: {log_id}")
        else:
            None
elif script_type == "replay":
        if log_id:
            logger.info(f"Replay ID: {log_id}")
        else:
            None
```